README.md

```diff
@@ -57,120 +60,81 @@ attach an event listener after the `deviceready` event fires.
 - BlackBerry 10
 - Windows Phone 7 and 8
 - Windows (Windows Phone 8.1 only)
-- Tizen
 - Firefox OS
-- Browser
+- Browsers
+    - Chrome
+    - Firefox
+    - Opera
 
-### Android and Amazon Fire OS Quirks
+### Quirks: Android &amp; Amazon Fire OS
 
-- Warning: the Android + Fire OS implementations are greedy and prolonged use will drain the user's battery. 
+**Warning**: the Android and Fire OS implementations are greedy and prolonged use will drain the device's battery.
 
-### Windows Phone 7 and 8 Quirks
+### Quirks: Windows Phone 7 &amp; Windows Phone 8
 
-Windows Phone 7 does not provide native APIs to determine battery
-level, so the `level` property is unavailable.  The `isPlugged`
-parameter _is_ supported.
+The `level` property is _not_ supported on Windows Phone 7 because the OS does not provide native APIs to determine battery level. The `isPlugged` parameter _is_ supported.
 
-### Windows Quirks
+### Quirks: Windows &amp; Windows Phone 8.1
 
-Windows Phone 8.1 does not support `isPlugged` parameter.
-The `level` parameter _is_ supported.
+The `isPlugged` parameter is _not_ supported on Windows Phone 8.1. The `level` parameter _is_ supported.
 
-### Browser Quirks
+## batterylow event
 
-Supported browsers are Chrome, Firefox and Opera. 
+Fires when the battery charge percentage reaches the low charge threshold. This threshold value is device-specific. Returns an [object][status_object] containing battery status.
 
 ### Example
 
-    window.addEventListener("batterystatus", onBatteryStatus, false);
+    window.addEventListener("batterylow", onBatteryLow, false);
 
-    function onBatteryStatus(info) {
-        // Handle the online event
-        console.log("Level: " + info.level + " isPlugged: " + info.isPlugged);
+    function onBatteryLow(status) {
+        alert("Battery Level Low " + status.level + "%");
     }
 
-## batterycritical
-
-The event fires when the percentage of battery charge has reached the
-critical battery threshold. The value is device-specific.
-
-The `batterycritical` handler is passed an object that contains two
-properties:
-
-- __level__: The percentage of battery charge (0-100). _(Number)_
-
-- __isPlugged__: A boolean that indicates whether the device is plugged in. _(Boolean)_
-
-Applications typically should use `window.addEventListener` to attach
-an event listener once the `deviceready` event fires.
-
 ### Supported Platforms
 
 - Amazon Fire OS
 - iOS
 - Android
 - BlackBerry 10
-- Tizen
 - Firefox OS
 - Windows (Windows Phone 8.1 only)
-- Browser
+- Browsers
```

Comment:
Browser is the name of the platform. For the supported ones, maybe just list them in parentheses like windows does above